string problem
string[] dir
---
bool status
